src/Parser/Druid/Restoration/Modules/Features/Clearcasting.js

```diff
@@ -4,125 +4,149 @@ import { formatPercentage } from 'common/format';
+        return suggest(<Wrapper>Your <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} /> procs should be used quickly so they do not get overwritten or expire.</Wrapper>)
           .icon(SPELLS.CLEARCASTING_BUFF.icon)
-          .actual(`You missed ${(this.totalCCProccs - this.used)}/${(this.totalCCProccs)} procs`)
-          .recommended(`<${Math.round(formatPercentage(recommended))}% is recommended`)
-          .regular(SuggestionThresholds.MISSED_CLEARCASTS.regular).major(SuggestionThresholds.MISSED_CLEARCASTS.major);
+          .actual(`You missed ${this.wastedProcs} out of ${this.totalProcs} (${formatPercentage(1 - this.clearcastingUtilPercent, 1)}%) of your free regrowth procs`)
+          .recommended(`<${Math.round(formatPercentage(1 - recommended, 1))}% is recommended`);
       });
-
-    const percentNonCCRegrowths = this.nonCCRegrowths / this.totalRegrowths;
-
-    when(percentNonCCRegrowths).isGreaterThan(SuggestionThresholds.NON_CC_REGROWTHS.minor)
+    when(this.nonCCRegrowthsSuggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span><SpellLink id={SPELLS.REGROWTH.id} /> is an inefficient spell to cast without a <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} /> proc.</span>)
+        return suggest(<Wrapper><SpellLink id={SPELLS.REGROWTH.id} /> is a very inefficient spell to cast without a <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} /> proc. It should only be cast when your target is about to die and you do not have <SpellLink id={SPELLS.SWIFTMEND.id} /> available.</Wrapper>)
           .icon(SPELLS.REGROWTH.icon)
-          .actual(`${formatPercentage(percentNonCCRegrowths)}% of your Regrowths were cast without a Clearcasting proc.`)
-          .recommended(`${Math.round(formatPercentage(recommended))}% is recommended`)
-          .regular(SuggestionThresholds.NON_CC_REGROWTHS.regular).major(SuggestionThresholds.NON_CC_REGROWTHS.major);
+          .actual(`You cast ${this.nonCCRegrowthsPerMinute.toFixed(1)} Regrowths per minute without a Clearcasting proc.`)
+          .recommended(`${recommended.toFixed(1)} CPM is recommended`);
       });
   }
 
   statistic() {
     return (
       <StatisticBox
         icon={<SpellIcon id={SPELLS.CLEARCASTING_BUFF.id} />}
-        value={`${formatPercentage(this.unusedClearcastingPercent)} %`}
-        label="Unused Clearcasts"
-        tooltip={`You got <b>${this.totalCCProccs} Clearcasting procs</b> and <b>used ${this.used}</b> of them.
-            <b>${this.nonCCRegrowths} of your Regrowths were used without a Clearcasting proc</b>.
+        value={`${formatPercentage(this.clearcastingUtilPercent, 1)} %`}
+        label="Clearcasting Util"
+        tooltip={`Clearcasting proced <b>${this.totalProcs} free Regrowths</b>
+            <ul>
+              <li>Used: <b>${this.usedProcs}</b></li>
```

Comment:
Small spelling of ``proced``, should be ``procced``

Reply:
Neither of them are words! Sometimes I was often spelling it 'proced'. I guess I could match style of rest of analyzer lol